rootfs.sh

```diff
@@ -114,13 +114,18 @@ cd tmp
 find . | cpio -H newc -o | gzip > initramfs.igz
 
 # Run
-${QEMU_CMD:-qemu-system-x86_64} -enable-kvm \
-    -m "${RAM:-8G}" \
-    -nographic \
-    -no-reboot \
-    -kernel ../../kernel -initrd initramfs.igz -append console=ttyS0
+if [ "$QEMU_CMD" = 'chroot' ]; then
+	PATH="/after/bin:$PATH" sudo chroot . /init
```

Comment:
maybe swap `sudo` and `PATH`
```suggestion
	sudo PATH="/after/bin:$PATH" chroot . /init
```